libnetdata/libnetdata.c

```diff
@@ -1492,3 +1492,35 @@ char *read_by_filename(char *filename, long *file_size)
         *file_size = size;
     return contents;
 }
+
+char *find_and_replace(const char *src, const char *find, const char *replace, const char *where)
+{
+   size_t size        = strlen(src) + 1;
+   size_t find_len    = strlen(find);
+   size_t repl_len    = strlen(replace);
+   char *value, *dst;
+
+   if (likely(where))
+       size += (repl_len - find_len);
+
+   value = mallocz(size);
+   dst = value;
+
+   if (likely(where)) {
+       size_t count = where - src;
+
+       dst = value + (dst - value);
```

Comment:
`(dst - value)` is always zero.

Reply:
You're right, of course, I removed the assignment since it's already done before the if. Thanks!